recipe/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [[ $target_platform == linux-aarch64 ]]; then
+    cmake   -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
We should be explicit about the build type (Release) and we should also pass `$CMAKE_ARGS`.